README.md

```diff
@@ -87,23 +105,23 @@ JDBC URL syntax
 
 The driver recognises JDBC URL's of the form:
 
-	jdbc:postgresql:database
+	jdbc:postgresqllegacy:database
```

Comment:
Changed the jdbc connection string identifier so it can co-exist with newer driver. 
However, we need to figure out a deployment / migration plan for existing data sources as they have to be updated to this new id with this new driver.